### PR TITLE
Implement Basic GUI.

### PR DESCRIPTION
**Pull request by jessetilro**
Implement basic GUI which allows users to enter the IP-Address of the Game Server and connect. I threw away the entire jMonkeyEngine application, AndroidHarness activity et cetera because it is simply to bloated and inefficient. Network API of jMonkeyEngine is still put to good use in this architecture.

### CODE REVIEW

File: app/src/main/java/nl/tudelft/pixelperfect/client/ConnectTask.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import com.jme3.network.serializing.Serializer;
+
+import java.io.IOException;
+
+/**
+ * Created by jesse on 10-5-2016.
+ */
+public class ConnectTask extends AsyncTask<String, Void, Client> {
+
+    private Exception exception;
```
Comment (wouterz) [inline app/src/main/java/nl/tudelft/pixelperfect/client/ConnectTask.java:16]: this seems unused

File: app/src/main/java/nl/tudelft/pixelperfect/client/ConnectTask.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    public ConnectResponse delegate;
+
+    protected Client doInBackground(String... ip) {
```
Comment (wouterz) [inline app/src/main/java/nl/tudelft/pixelperfect/client/ConnectTask.java:19]: what does (String... ip) mean|? I've never seen this before

**Comment by dalderliesten** [comment]
Agreed with Wouter's comments, nothing else to add.

File: app/src/main/java/nl/tudelft/pixelperfect/client/ConnectTask.java (hunk 2 above)
Comment (wouterz) [inline app/src/main/java/nl/tudelft/pixelperfect/client/ConnectTask.java:19]: apparently this is a [vararg](http://stackoverflow.com/questions/3158730/java-3-dots-in-parameters), cool.

File: app/src/main/java/nl/tudelft/pixelperfect/client/ConnectTask.java (hunk 1 above)
Comment (jessetilro) [inline app/src/main/java/nl/tudelft/pixelperfect/client/ConnectTask.java:16]: Will refactor the code and resolve static errors in upcoming PR.

**Pull request merged**